lighthouse-core/audits/byte-efficiency/byte-efficiency-audit.js

```diff
@@ -161,6 +175,8 @@ class UnusedBytes extends Audit {
       wastedMs,
       wastedBytes,
     };
+
+    // @ts-ignore - TODO(bckenny): unify details types. items shouldn't be an indexed type.
```

Comment:
we need a unified theory of details item types rather than some with `{[x: string]: DetailsItem}` and some not (if it doesn't have a generic index you can't assign to a type that does). I'd rather punt here than figure that out, though, since we're just going to have to figure it out all over again when we switch to our new details types